iroh/src/magicsock/relay_actor.rs

```diff
@@ -184,6 +175,12 @@ impl ActiveRelayActor {
         // the same time.
         let mut relay_send_fut = std::pin::pin!(MaybeFuture::none());
 
+        // If inactive for one tick the actor should exit.  Inactivity is only tracked on
+        // the last datagrams sent to the relay, received datagrams will trigger ACKs which
+        // is sufficient to keep active connections open.
+        let mut inactive_timeout = tokio::time::interval(RELAY_INACTIVE_CLEANUP_TIME);
+        inactive_timeout.reset(); // skip immediate tick
```

Comment:
wouldn't using `interval_at` be cleaner? (honest question, not sure it does what you need here)

Reply:
probably not, gets very verbose again and then i'd prefer the original sleep probably.  going to leave this as-is.